projects/jdwp/defs/command_sets/virtual_machine.py

```diff
@@ -54,11 +54,33 @@
     },
 )
 
+__Version_reply = Struct(
+    [
+        Field("description", Type.STRING, "Text information on the VM version"),
+        Field("jdwpMajor", IntegralType.INT, "JDWP major version number"),
+        Field("jdwpMinor", IntegralType.INT, "JDWP minor version number"),
+        Field(
+            "vmVersion",
+            Type.STRING,
+            "Target VM JRE version, as in the java.version property",
+        ),
+        Field("vmName", Type.STRING, "Target VM name, as in the java.vm.name property"),
+    ]
+)
+
+Version = Command(
+    name="Version",
+    id=2,
+    out=None,
+    reply=__Version_reply,
+    error={ErrorType.VM_DEAD},
+)
 
 VirtualMachine = CommandSet(
     name="VirtualMachine",
     id=1,
     commands=[
         AllClasses,
+        Version,
```

Comment:
Could we keep this sorted by command's id ?

Reply:
I have changed the order, by IDs of commands in the command set.